server/app/crud/crud.py

```diff
@@ -64,18 +71,45 @@ def delete_metadata(key):
     connection= create_connection(db_file)
     cursor= connection.cursor()
     cursor.execute("""
-    Delete from store where key=? 
+    Delete from store where generated_key=? 
                    """,(key,))
     connection.commit()
     cursor.close()
 
 def check_key_existence(key):
         connection = create_connection(db_file)
         cursor = connection.cursor()
-        cursor.execute("SELECT id FROM store WHERE key=?", (key,))
+        cursor.execute("SELECT id FROM store WHERE generated_key=?", (key,))
         result = cursor.fetchone()
         connection.close()
         if result is None:
             return False
         else:
-            return True
\ No newline at end of file
+            return True
+
+
+def get_latest_generated_key():
+    try:
+        connection = create_connection(db_file)
+        cursor = connection.cursor()
+        cursor.execute("""
+        SELECT key FROM generated_keys ORDER BY generated_at DESC LIMIT 1;
```

Comment:
Let'sl select the last value from store instead

Reply:
hi, @Tejas-ChandraShekarRaju 
Currently there is no timestamp in the store table though,
Can we use id for sorting?
Or would it be better to add a type stamp and sort by that?